### ISSUE TITLE
Remove vendor-specific info from resource usage monitoring

### ISSUE THREAD

**Issue by zacharysarah** [kubernetes/website#15711]
Related to #15576 

The [resource usage monitoring](https://k8s.io/docs/tasks/debug-application-cluster/resource-usage-monitoring.md) page contains vendor-specific content that does not meet K8s style guide standards. (See #15576) At most, this page should mention third-party solutions with a link. Any PR that addresses this issue should consider refactoring solutions into a simple hyperlinked list.

TLDR: Kubernetes docs are not the place for vendor pitches. 

/good-first-issue
@aimeeu

**Comment by sftim** [comment]
@zacharysarah https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (see current PR description) is a broken link BTW
Think you mean https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/
